lib/obsolete.gd

```diff
@@ -55,6 +55,17 @@
 ##  Usually, old names will be available for several releases;
 ##  they may be removed when they don't seem to be used any more.
 ##  <P/>
+##  Information about obsolete names is printed by <Ref Func="Info"/> using the
+##  InfoObsolete InfoClass. By default InfoObsolete is set to 1. Newly
+##  obsoleted identifiers should be first be outputted at info level 2. Once they
+##  have been removed from all packages, they should then be moved to info level
+##  1, so they are visible to normal users, for at least one major release before
+##  being removed.
+##  </P>
+##  The functions DeclareObsoleteSynonym and DeclareObsoleteSynonymAttr take
+##  an optional final paremeter, specifying the Info level at which they print,
```

Comment:
"specifying the info level at which the given obsolete symbol should be reported. It defaults to 1."